11-payment-encoding.md

```diff
@@ -55,23 +59,25 @@ The following `multiplier` letters are defined:
 ## Requirements
 
 A writer:
-  - MUST encode `prefix` using the currency it requires for successful payment
-  - If it requires a specific minimum amount for successful payment:
-	- MUST include that `amount`
-	- MUST encode `amount` as a positive decimal integer with no leading 0s
-	- SHOULD use the shortest representation possible by using the largest
-	  multiplier or omitting the multiplier
+  - MUST encode `prefix` using the currency required for successful payment.
+  - if a specific minimum `amount` is required for successful payment:
+	  - MUST include that `amount`.
+	- MUST encode `amount` as a positive decimal integer with no leading 0s.
+	- SHOULD use the shortest representation possible, by using the largest
+	  multiplier or omitting the multiplier.
 
 A reader:
-  - MUST fail if it does not understand the `prefix`
-  - If the `amount` is empty:
-	- SHOULD indicate if amount is unspecified
-  - Otherwise:
-	- MUST fail if `amount` contains a non-digit or is followed by
-      anything except a `multiplier` in the table above
-    - If the `multiplier` is present:
-	  - MUST multiply `amount` by the `multiplier` value to derive the
-        amount required for payment
+  - if it does not understand the `prefix`:
+    - MUST fail the payment. 
+  - if the `amount` is empty:
+	  - SHOULD indicate that amount is unspecified [FIXME: indicate where or to whom?].
```

Comment:
- SHOULD indicate to the payer that amount is unspecified

Reply:
...and done. No worries, you're a busy man!